apps/showcase/eslint.local.config.mjs

```diff
@@ -67,6 +67,12 @@ export default [
             }
           }
         }
+      ],
+      '@o3r/o3r-restriction-key-tags': [
+        'error',
+        {
+          supportedKeys: ['restriction_1', 'restriction 2']
```

Comment:
I know this is to show the different possibilities but it looks a bit weird to have one with space and the other with underscore
Maybe we can make it more obvious like `restriction_with_underscores`, `restriction with spaces`

Reply:
as discussed changed for `"api owners"`

Reply:
same comment for the doc file